src/core_plugins/metrics/server/lib/vis_data/calculate_indices.js

```diff
@@ -37,7 +37,11 @@ function calculateIndices(req, indexPattern = '*', timeField = '@timestamp', off
   const { callWithRequest } = server.plugins.elasticsearch.getCluster('data');
   const params = getParams(req, indexPattern, timeField, offsetBy);
   return callWithRequest(req, 'fieldStats', params)
-    .then(handleResponse);
+    .then(handleResponse)
+    .then(indices => {
```

Comment:
minor: consider merging this into `handleResponse`. It seems the only reason the `calculate_indices` module exports handleResponse is so it can be unit-tested. By chaining this additional fallback to it, the unit test has gotten somewhat stale.